wiki/en/Software-Manual.md

```diff
@@ -84,7 +84,7 @@ The Connection Setup window shows a list of available servers together with the
 
 You can filter the list by server name or location. To list only occupied servers, enter a "#" character.
 
-If you know the IP address or URL of a private server, you can connect to it using the Server Name/Address
+If you know the IP address or URL of unregistered server, you can connect to it using the Server Name/Address
```

Comment:
Line 83, above this
> The Connection Setup window shows

...which it isn't.  It's the "Connect" window.  The section probably needs splitting into the separate fields more clearly.

Reply:
Not sure I understand. The window title says "Connection Setup".

Reply:
On Windows, I see:
![image](https://user-images.githubusercontent.com/1549463/148693494-b8aa8e58-597f-461c-b487-066eb39d8ecf.png)

The window name is "Connect" there.